README.md

```diff
@@ -10,36 +10,38 @@
 
 Try .NET provides developers and content authors with tools to create interactive experiences.
 
-There are different flavors of  Try .NET interactive experiences, from the web experience powered by Blazor (as seen on [Microsoft docs](https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/intro-to-csharp/hello-world)), to interactive documentation for .NET Core with the [dotnet try global tool](DotNetTry.md), to [.NET Jupyter Notebooks](Notebook.md). 
+There are different experiencs of  Try .NET interactive experiences, from the web experience powered by Blazor (as seen on [Microsoft docs](https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/intro-to-csharp/hello-world)), to interactive documentation for .NET Core with the [dotnet try global tool](DotNetTry.md). 
 
-## Flavors of Try .NET 
+## Experiences  
```

Comment:
We can probably do without this header, and maybe even the two bullet points I suggested, since the following `###` headings reiterate them.